packages/config/config/devices/0x0129/yrm276.json

```diff
@@ -129,7 +123,6 @@
 		},
 		"8": {
 			"label": "Operating Mode",
-			"description": "Normal (default)\nVacation = keypad lockout\nPrivacy = no keypad, RF Unlock will work",
```

Comment:
I think the description is okay - it offers additional info what the options do.